components/_cover-photo.scss

```diff
@@ -73,6 +77,6 @@
 .cover-photo-cta {
   margin-top: 1.0em;
   @include padding(1.75em 1.5em);
-  width: 80%;
+  width: 50%;
```

Comment:
this is making things render funny on my monitor. text shouldn't be limited to a ~400px wide box on a 1440p screen. Test on something larger than an MBA and smaller AND larger than an iPhone 6/S6 size display.